plugin/src/main/java/org/jfrog/idea/xray/utils/Utils.java

```diff
@@ -152,4 +145,40 @@ public static TreeExpansionListener getIssuesTreeExpansionListener(TreeExpansion
         }
         return null;
     }
+
+    /**
+     * Returns Set of Paths cleaned of subdirectories.
+     * For example the set ["/a", "/b/c", "/a/d"] will become ["/a", "/b/c"]
+     * @param projectPaths Paths to filter
+     * @return Set of Paths cleaned of subdirectories.
+     */
+    public static Set<Path> filterProjectPaths(Set<Path> projectPaths) {
+        Set<Path> finalPaths = new HashSet<>();
+        Comparator<Path> compByPathLength = Comparator.comparingInt(pathA -> pathA.toString().length());
+        // Create a sorted by length list of paths
+        List<Path> sortedList = projectPaths.stream().sorted(compByPathLength).collect(Collectors.toList());
+        projectPaths.forEach(pathToExam -> {
+                    Iterator iterator = sortedList.iterator();
+                    boolean isARootPath = true;
```

Comment:
Let's rename **isARootPath** to **isRootPath**.
Also, the loop is not properly indented.